js/about.js

```diff
@@ -48,6 +48,10 @@ resizeInfoChunks();
 
 window.onresize = resizeInfoChunks;
 
+var currentPage = document.getElementById('navabout');
```

Comment:
This code definitely works, but I have some thoughts for improvement: rather than write the same code 5 times, let's make a single JavaScript file that this works with, and you can just import that same file on every page.

Then, do some branching based on the last portion of the URL – you could make a map that looks something like

```JavaScript
{
    '/about': 'navabout'
...
}
```

and if the current page URL lines up with something in the map, highlight the relevant nav link based on id.

Also, rather than manually define the color and fontweight, let's just make a class in the CSS and update the classList attribute of the relevant nav link.

Reply:
alright, working on it